src/Autocomplete.js

```diff
@@ -57,7 +65,19 @@ class Autocomplete extends Component {
   }
 
   _onChange (evt) {
-    this.setState({ value: evt.target.value });
+    const { onChange } = this.props;
+    const value = evt.target.value;
+    if (onChange) { onChange(evt, value); }
+
+    this.setState({ value });
+  }
+
+  _onAutocomplete (value, evt) {
+    const { onChange, onAutocomplete } = this.props;
+    if (onAutocomplete) { onAutocomplete(value); }
+    if (onChange) { onChange(evt, value); }
```

Comment:
This method is for when you click on an element from the list right? why are we calling `onChange` also?

Reply:
I was wondering about it. Technically the value gets changed so we notify the user about it.
It will also allow us to rely only on onChange/value props and not use onAutocomplete if you don't care that it was autocompleted.

Otherwise you'd have to use both every time.